packit/base_git.py

```diff
@@ -444,13 +453,12 @@ def download_remote_sources(self):
             local_file = Path(self.specfile.sources_location).joinpath(basename)
             if not Path(local_file).is_file():
                 logger.info(f"File {local_file} doesn't exist locally, downloading it.")
-                url = self.get_matching_source_url_from_config(basename) or remote_file
-                logger.info(f"Downloading file from URL {url}. ")
+                logger.info(f"Downloading file from URL {remote_file}.")
                 try:
-                    DownloadHelper.download_file(url, str(local_file))
+                    DownloadHelper.download_file(remote_file, str(local_file))
                 except DownloadError as e:
                     raise RebaseHelperError(
-                        f"Failed to download file from URL {url}. Reason: {e}. "
+                        f"Failed to download file from URL {remote_file}. Reason: {e}."
                     ) from e
```

Comment:
as the downloading of the sources from our config file is separated, this can be replaced by the original [rebase-helper download_remote_sources() method](https://github.com/rebase-helper/rebase-helper/blob/0be4d04b195d46f3880107e7f6b7e888733f59c5/rebasehelper/specfile.py#L144)

Reply:
interesting, so we just implemented a different way for CentOS' lookaside